app/gimlet/rev-b.toml

```diff
@@ -232,7 +243,7 @@ uses = [
     "system_flash", # TODO also used by `net`, both to read the stm32 uid
 ]
 task-slots = ["jefe", "net", "update_server", "sys"]
-features = ["gimlet", "usart1", "vlan"]
+features = ["gimlet", "usart1", "vlan", "baud_rate_3_000_000", "hardware_flow_control"]
```

Comment:
Nit: maybe `baud_rate_3M` instead?  Or does that cause compiler complaints about not being `snake_case`?

Reply:
I get no complaints with `baud_rate_3M` :+1: